evm_loader/program/src/account_data.rs

```diff
@@ -219,10 +219,10 @@ impl AccountData {
     ///
     /// Will return:
     /// `ProgramError::InvalidAccountData` if doesn't contain `Account` struct
-    pub fn get_account(&self) -> Result<&Account, ProgramError>  {
+    pub const fn get_account(&self) -> Result<&Account, ProgramError>  {
         match self {
             Self::Account(ref acc) => Ok(acc),
-            _ => Err!(ProgramError::InvalidAccountData),
+            _ => Err(ProgramError::InvalidAccountData),
```

Comment:
Why is it necessary to disable the use of this macro?

Reply:
It is not necessary. I reverted this file.